sdk/communication/Azure.Communication.CallingServer/CHANGELOG.md

```diff
@@ -1,15 +1,9 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.0.0-beta.2 (2021-07-23)
```

Comment:
you would need to make a corresponding version change in the readme as well.

Reply:
Updated